### ISSUE TITLE
Fill gaps in Hestia to ecoivent biosphere mapping

### ISSUE THREAD

**Issue by TimoDiepers** [brightway-lca/bw_hestia_bridge#50]
So far, we only have a preliminary version for the milestone. There remain some gaps that need to be filled for Hestia flows that don't have an ecoivent equivalent.

**Comment by tfardet** [comment]
Does #56 change anything about this issue? (I expect not, but just checking)

**Comment by cmutel** [comment]
@tfardet No, this helps but there are flows labelled in other groups which could also be matched the to the ecoinvent flow list.
